var/spack/repos/builtin/packages/xrootd/package.py

```diff
@@ -76,6 +76,11 @@ def cmake_args(self):
             format('ON' if '+readline' in spec else 'OFF'),
             '-DENABLE_CEPH:BOOL=OFF'
         ]
+        # see https://github.com/spack/spack/pull/11581
+        if '+python' in self.spec:
+            options.append('-DPYTHON_EXECUTABLE=%s/python' %
+                           self.spec['python'].prefix.bin)
```

Comment:
Instead use `spec['python'].command.path`. This works regardless of whether the executable is called `python` or `python3`.